.github/workflows/pypi-publish.yml

```
@@ -0,0 +1,46 @@
+name: Publish distributions to TestPyPI and PyPI
+
+# Only activate if the v* tag is pushed.
+on:
+  push:
+    tags:
+    - v*
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to TestPyPI and PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+
+    - name: Install twine
+      run: >-
+        python -m pip install -U twine
+
+    - name: Build a tar ball
+      run: >-
+        python setup.py sdist
+
+    - name: Verify the distributions
+      run: twine check dist/*
+
+    - name: Publish distribution to Test PyPI
+      # Do not upload the package from forked repository.
```

Is it correct that the Github repository constrains to be `optuna/optuna`, so we can't automatically upload from forked repositories? If so, I think it would be appropriate to make this comment less imperative, as follows.
```suggestion
      # The following upload action cannot be executed in the forked repository.
```

---

Thank you for your suggestion. That makes sense. I'll also fix the same comment in `Publish distribution to PyPI`.